docs/tutorial/src/mygc/ss/collection.md

```diff
@@ -132,6 +132,17 @@ scheduler's prepare stage and resumes the mutators. The `StopMutators` work
 will invoke code from the bindings to scan threads and other roots, and those 
 scanning work will further push work for a transitive closure.
 
+Though you can add those work packets by yourself, `CommonPlan` provides a
```

Comment:
It is now `Scheduler.schedule_common_work`.  There are several other places mentioning `schedule_common` in this file.